### ISSUE TITLE
Accessibility regression in the prefix/suffix feature of number fields

### ISSUE THREAD

**Issue by sergei-maertens** [open-formulieren/formio-renderer#244]
**Legacy renderer**

Reference: https://open-formulieren.github.io/open-forms-sdk/?path=/story/form-io-components-vanilla-number--number-with-suffix

The input has `aria-labelledby` pointing to label ID & suffix ID, which is the expected accessible behaviour for screen readers.

**Own renderer**

There are no `aria-labelledby` IDs specified at all. These should be specified.

The fix is probably something in the `InputContainer` element itself and if possible, it'd be great if this is something developers don't need to worry about when using these components, so it should ideally be enforced through the type system.
